### PR TITLE
Update content atom version to 0.2.6

### PR DESCRIPTION
**Pull request by tomrf1**
Corresponding content-atom changes:
https://github.com/guardian/content-atom/pull/12/files
https://github.com/guardian/content-atom/pull/19/files

### CODE REVIEW

**Comment by mchv** [comment]
Idem as https://github.com/guardian/content-api/pull/1373#issuecomment-182820796

**Comment by mchv** [comment]
Please update the changelog as well and fill the PR description :)

**Comment by tomrf1** [comment]
@mchv is it ok to release this now?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 7.27
+* Update content-atom version to 0.2.6
```
Comment (LATaylor-guardian) [inline CHANGELOG.md]: The change log should reflect the actual changes.

**Comment by mchv** [comment]
:+1:  worth merging as well [story-packages](https://github.com/guardian/content-api-scala-client/pull/133) and release @cb372 [fix of tweet element](https://github.com/guardian/content-api-scala-client/pull/134)

**Comment by tomrf1** [comment]
I believe the story-packages update can be merged any time, but we should
be careful with this content-atom (quiz) change since it defines the id
fields as mandatory and we currently have quizzes in PROD ES with no ids.

On 15 February 2016 at 18:14, Mariot Chauvin <notifications@github.com>
wrote:

> [image: :+1:] worth merging as well story-packages
> <https://github.com/guardian/content-api-scala-client/pull/133> and
> release @cb372 <https://github.com/cb372> fix of tweet element
> <https://github.com/guardian/content-api-scala-client/pull/134>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/content-api-scala-client/pull/131#issuecomment-184331869>
> .
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

**Pull request merged**